docs/sources/visualizations/dashboards/manage-dashboards/index.md

```diff
@@ -144,6 +144,14 @@ To restore one or more dashboards, follow these steps:
    - If the original folder no longer exists, you’ll need to select a new target folder.
 4. Click **Restore**.
 
+### Limitations
+
+When you restore dashboards, be aware of the following limitations:
+
+- **Permissions aren't preserved**: Dashboard-specific permissions are not restored. After restoration, you'll need to manually reconfigure any custom permissions that were previously set on the dashboard.
+- **Folder-level permissions apply**: Restored dashboards inherit the permissions of the target folder you select during restoration.
```

Comment:
Because of the way the first bullet is worded, this one can sound like we mean that the previously existing folder permissions apply. Tweaked the wording slightly:
 
```suggestion
- **Folder-level permissions are applied** - Restored dashboards inherit the permissions of the target folder you select during restoration.
```